tracee-ebpf/tracee/events_processor.go

```diff
@@ -197,11 +197,6 @@ func (t *Tracee) processEvent(ctx *context, args map[string]interface{}, argMeta
 					*argMetas = append(*argMetas, hashMeta)
 					ctx.Argnum += 1
 					args["sha256"] = currentHash
-
-					ctimeMeta := external.ArgMeta{"ctime", "unsigned long"}
-					*argMetas = append(*argMetas, ctimeMeta)
-					ctx.Argnum += 1
-					args["ctime"] = sourceFileCtime
```

Comment:
With this change in place, there is no more reason to have sourceFileCtime and even sourceFileStat is not required anymore, right? If so, can we just remove those and replace it by the ctime argument of the sched_process_exec event?